tests/unit/hive_metastore/test_locations.py

```diff
@@ -46,6 +46,20 @@ def test_list_mounts_should_return_a_deduped_list_of_mount_without_encryption_ty
     assert expected == backend.rows_written_for("hive_metastore.test.mounts", "append")
 
 
+def test_list_mounts_should_return_a_deduped_list_of_mount_without_variable_volume_names():
+    mounts = [
+        Mount(name="/Volume", source="DbfsReserved"),
+        Mount(name="/Volumes", source="DbfsReserved"),
```

Comment:
Can we just filter out dbfs reserved mounts instead?

Reply:
checked for DbfsReserved.